app/models/admin/FeatureFlags.scala

```diff
@@ -21,110 +21,115 @@ import uk.gov.hmrc.mongoFeatureToggles.model.{Environment, FeatureFlagName}
+  override val name: String                = "tax-components-retrieval-toggle"
   override val description: Option[String] = Some(
-    "Enable/disable calls for tax-components from tai (NPS). Used for marriage allowance and high income child benefit charge"
+    "Enable retrieval of tax components from TAI (NPS), used for displaying Marriage Allowance and High Income Child Benefit Charge"
   )
+  override val defaultState: Boolean = true
 }
 
 case object RlsInterruptToggle extends FeatureFlagName {
   override val name: String                = "rls-interrupt-toggle"
-  override val description: Option[String] = Some("Enable/disable the interrupt for return letter to sender (RLS)")
+  override val description: Option[String] = Some(
+    "Show interrupt page if a residential or correspondence address has RLS status and has not been updated"
+  )
 }
 
-case object PaperlessInterruptToggle extends FeatureFlagName {
+case object EnforcePaperlessPreferenceToggle extends FeatureFlagName {
   override val name: String                = "enforce-paperless-preference"
-  override val description: Option[String] = Some("Enable/disable the interrupt for paperless setting")
+  override val description: Option[String] = Some(
+    "Show an interrupt to prompt users to set or confirm their paperless communication preference"
+  )
 }
 
 case object TaxSummariesTileToggle extends FeatureFlagName {
   override val name: String                = "tax-summaries-tile"
-  override val description: Option[String] = Some("Enable/disable the tile for annual tax summary")
+  override val description: Option[String] = Some(
+    "Enable/disable the display of the Annual Tax Summary tile on the home page"
+  )
 }
 
 case object ShowPlannedOutageBannerToggle extends FeatureFlagName {
   override val name: String                = "show-outage-banner-toggle"
   override val description: Option[String] = Some(
-    "Enable/disable banner warning users of a planned outage. Banner content to be adjusted in app-config first"
+    "Show or hide the planned outage banner on key pages. Banner content is configured via app-config"
   )
 }
 
 case object AgentClientRelationshipsToggle extends FeatureFlagName {
   override val name: String                = "agent-client-relationships-toggle"
   override val description: Option[String] = Some(
-    "Enable/disable calls to agent-client-relationships service from Profile and Setting page"
+    "Enable/disable the check for agent-client relationships on the Profile and Settings page"
   )
 }
 
 case object BreathingSpaceIndicatorToggle extends FeatureFlagName {
   override val name: String                = "breathing-space-indicator-toggle"
   override val description: Option[String] = Some(
-    "Enable/disable calls to Breathing space"
+    "Enable/disable the call to Breathing Space service to determine protected debt period status for the user"
   )
 }
 
-case object GetPersonFromCitizenDetailsToggle extends FeatureFlagName {
```

Comment:
This toggle is not currently called. But it probably should be -looks like the one place where citizen details is called is not using this flag, so probably should reinstate this flag and call it: https://github.com/hmrc/pertax-frontend/blob/64733811a131fec001298f8953038911c139c218/app/controllers/auth/SelfAssessmentStatusAction.scala#L59